src/cluster.cr

```diff
@@ -0,0 +1,20 @@
+class Cluster
+
+    def self.fork (env : Hash)
+        env["FORKED"] = "1"
+        return Process.fork { Process.run(PROGRAM_NAME, nil, env, true, false, true, true, true, nil ) }
+    end
+
+    def self.isMaster
```

Comment:
NodeJS API adheres to JS conventions, not to Ruby ones. It's a convention to use `isFoo` in JS, whereas in Ruby you'd rather write `foo?`.